src/components/Cards.js

```diff
@@ -1,22 +1,35 @@
 export default class Card {
-  constructor(data, cardSelector, handleImageClick) {
+  constructor(
+    data,
+    cardSelector,
+    handleImageClick,
+    deleteCard,
+    likeCard,
+    unlikeCard
+  ) {
     this.name = data.name;
     this.link = data.link;
     this._cardSelector = cardSelector;
     this._handleImageClick = handleImageClick;
+    this._deleteCard = deleteCard;
+    this._id = data._id;
+    this._isLiked = data.isLiked;
+    this._likeCard = likeCard;
+    this._unlikeCard = unlikeCard;
   }
 
-  _setEventListeners() {
+  setEventListeners() {
     this._likeButton = this._cardElement.querySelector(".card__like-button");
+    this._likeButton.classList.toggle("card__like-button_active");
```

Comment:
`    this._likeButton.classList.toggle("card__like-button_active");
`

Please, delete it

* New cards appear already **liked**, but the heart should be `white` initially